packages/web/src/components/spa/AboutSection/AboutSection.jsx

```diff
@@ -10,17 +10,14 @@ export const AboutSection = () => {
             <div className="about-text">
                 <h1 className="about-title">About us</h1>
```

Comment:
The about us picture is too big and there is an effect on it which makes everyone look like a cartoon. Please choose another picture.

Reply:
@SuelaCark on my laptop i dont seem to see an effect on it, but i will change the picture to a different one. the ones we have to choose from are extremely similar, but hopefully there is a difference